### ISSUE TITLE
Invalid format string: array.access.unsafe.high.range

### ISSUE THREAD

**Issue by panacekcz** [typetools/checker-framework#1984]
The error message for array.access.unsafe.high.range is an invalid format string. Probably `%d` is not allowed.

Code example:
```java
import org.checkerframework.common.value.qual.IntRange;

public class AccessUnsafeMessage {
	public int m(int[] a, @IntRange(from = 0, to = 12) int i) {
		return a[i];
	}
}
```
Output of the Index Checker:
```
AccessUnsafeMessage.java:5: error: Invalid format string: "[array.access.unsafe.high.range] Potentially unsafe array access: the index could be larger than the array's bound
		return a[i];
		         ^
  index type found: %s
  array type found: %s
  required        : index of type @IndexFor("%s") or @LTLengthOf("%s"), or array of type @MinLen(%d)" args: [@IntRange(from=0, to=12) int, @UnknownVal int @UnknownVal [], a, a, 13]
1 error
```
Expected:
```
AccessUnsafeMessage.java:5: error: [array.access.unsafe.high.range] Potentially unsafe array access: the index could be larger than the array's bound
		return a[i];
		         ^
  index type found: @IntRange(from=0, to=12) int
  array type found: @UnknownVal int @UnknownVal []
  required        : index of type @IndexFor("a") or @LTLengthOf("a"), or array of type @MinLen(13)
1 error
```
